src/focus.service.ts

```diff
@@ -554,6 +554,13 @@ export class FocusService {
       return false;
     }
 
+    for (let parent = el; parent; parent = parent.parentElement) {
+      const record2 = this.registry.find(parent);
```

Comment:
IIRC our linter settings forbid shadowing (which is a good thing), should choose a different name tho.